### PR TITLE
devices: Netonix switch update

### PR DESCRIPTION
**Pull request by murrant**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Updated mib
Added total power consumption and dc power input (for dc switches)
Small cleanups
fixes: #4487

### CODE REVIEW

File: includes/discovery/mempools/netonix.inc.php [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+ * @link       http://librenms.org
+ * @copyright  2016 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
+ */
 
 if ($device['os'] == 'netonix') {
     echo 'NETONIX : ';
 
-    $free = str_replace('"', "", snmp_get($device, 'UCD-SNMP-MIB::memTotalFree.0', '-OvQU'));
+    $free = snmp_get($device, 'UCD-SNMP-MIB::memTotalFree.0', '-OvQU');
```
Comment (laf) [inline includes/discovery/mempools/netonix.inc.php]: How come the str_replace is being dropped?

Comment (murrant) [inline includes/discovery/mempools/netonix.inc.php]: It wasn't needed on my device machine, I'll test on another device.  I think it was just copy/paste before...

Comment (laf) [inline includes/discovery/mempools/netonix.inc.php]: kk, let me know. Good for a merge once checked.

Comment (murrant) [inline includes/discovery/mempools/netonix.inc.php]: Looks all good to me returns "77768\n", which is_numeric() == true

**Pull request merged**